src/include/connection.h

```diff
@@ -172,6 +172,24 @@ struct __wt_disaggregated_checkpoint_track {
     wt_timestamp_t timestamp;
 };
 
+/*
+ * WT_PAGE_DELTA --
+ *      Configuration for the page delta.
```

Comment:
Configurations?

Reply:
Fixed in 8f5b4ce4b3612e6dc75ff4fcb72d9392766a75b3